package.json

```diff
@@ -78,6 +77,8 @@
     "primer-support": "^5.0.0",
     "query-string": "^6.5.0",
     "require-dir": "^1.1.0",
+    "octicons": "^8.5.0",
+    "node-sass": "^4.12.0",
```

Comment:
Is the node-sass line needed?

Reply:
Needs this to handle version in a lock file for `node-sass-middleware`, without manually handling version of `node-sass` it's hard to updates to newer versions of Node. `node-sass-middleware` defaults version of `node-sass` it's `^4.9.0` (Node 8-10).